web_src/less/_base.less

```diff
@@ -1231,10 +1231,10 @@ i.icon.centerlock {
 
 .emoji,
 .reaction {
-    display: inline-block;
-    background-size: contain;
-    font-size: 1.5em !important;
-    vertical-align: middle !important;
+    font-size: 1.2em;
```

Comment:
@silverwind I liked this size better at 1.5em

Reply:
I copied that size from GitHub's definition, but feel free to revert. I did not test this yet. My thought is that if a page contains regular unicode emoji that are not rendered from tokens, those would not have the `.emoji` class and would render with a different size, but correct me if I'm wrong and the backend is able to detect those as well.

Reply:
GitHub seems to be able to detect unicode emoji and wrap them in a `<span>`:

From `:camel`: 🐫 
Pasted: 🐫

Reply:
I did make it able to detect all emoji and apply the class. I can't seem to make a working link to a line in a PR but it is emojiRegex inside modules/markup/html.go and emojiProcessor for that same file. Also I used this to test it https://regex101.com/r/XO3FWR/2

I like adjusting the size because its harder to see otherwise (many other sites adjust it from 1.2-1.5em as you've seen Github uses 1.2em I think gitlab uses 1.5em).

Reply:
If you already wrap existing unicode-emojis in markdown, we should be fine.

I'll test more later to form an opinion on the font size.

Reply:
Some patterns are not matched, or matched only partially (where adding `<span>` would break the composition semantings). Is this intentional, a know limitation, ...?

![image](https://user-images.githubusercontent.com/18600385/79034621-c42b5200-7b8d-11ea-8cf7-3300bc36697b.png)

Note: I've followed the test URL https://regex101.com/r/XO3FWR/2 as posted above.